spec/workers/recover_lost_jobs_worker_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe RecoverLostJobsWorker do
+  before do
+    Sidekiq::Worker.clear_all
+  end
```

Comment:
Minor, do we know if this block is needed? I don't think it actually does anything for the tests.

Reply:
It's only necessary when making assertions on things like queue size, etc. It can go. 👌